docs/apache-airflow/concepts/datasets.rst

```diff
@@ -93,7 +93,7 @@ If required, an extra dictionary can be included in a Dataset:
         extra={'team': 'trainees'},
     )
 
-..note::
+.. note ::
```

Comment:
For anyone curious: The whitespace _in front_ is significant; the one after is not.

Reply:
❤️ RST quirks